python/GafferUI/Backups.py

```diff
@@ -140,6 +141,16 @@ def recoveryFile( self, script ) :
 		if os.path.getmtime( backupFileName ) < os.path.getmtime( scriptFileName ) :
 			return None
 
+		ignorePatterns = [
+			# Commented lines, for instance a header.
+			r'#.*$',
+			# Version metadata.
+			r'Gaffer\.Metadata\.registerNodeValue\( parent, "serialiser:.*Version", .* \)$',
+			# Pesky catalogue port number, which is different each time we run.
+			r'parent\["variables"\]\["imageCataloguePort"\]\["value"\]\.setValue\( [0-9]* \)$',
```

Comment:
oof. this one is really specific... are we going to hit any others in production scenes?